Document resource identifiers (#394)
onflow/cadence: docs/language/composite-types.mdx

destroy resources
```

### Resource Identifier

> Resources have an impicit unique identifier associated with them,

Author: ```suggestion
Resources have an implicit unique identifier associated with them,
```

> all resources have a predeclared public field `let uuid: UInt64`.

Reviewer: ```suggestion
implemented by a predeclared public field `let uuid: UInt64` on each resource.
```
